plugins/firebase.js

```diff
@@ -13,6 +14,7 @@ if (!firebase.apps.length) {
 }
 
 export const db = firebase.database()
+export const firestoredb = firebase.firestore()
```

Comment:
Rename `firestoredb` as `DB`, as this will be the convention we will be proceeding